tests/unit/featurizers/test_cgr.py

```diff
@@ -381,5 +392,13 @@ def test_featurize_imbalanced(self, rxn_smi, mode_imbalanced):
         for i, bond in enumerate(bonds):
             expect_edge_index.extend([bond, bond[::-1]])
             expect_rev_edge_index.extend([i * 2 + 1, i * 2])
-        assert np.array_equal(molgraph.edge_index, expect_edge_index)
-        assert np.array_equal(molgraph.rev_edge_index, expect_rev_edge_index)
+
+        if len(expect_edge_index) == 0:
```

Comment:
this also felt unsatisfying - but this is basically because `molgraph.edge_index` is instantiated as an np array of `[ [] [] ]` (a list of at least two lists) whereas `expect_edge_index` is constructed by extending an empty list (`[]`). So in the null case, the molgraph edge will be an np array with 2 dimensions, but in the latter it will just be 1 dimensional, so they fail the `np._array_equal` test.

I welcome your feedback on how to write this more elegantly/cleanly if necessary!

Reply:
Maybe initiate it the same way as in `molgraph.edge_index`

Reply:
thanks! I changed it now.